backend/src/main/java/gov/cdc/usds/simplereport/db/model/DataHubUpload.java

```diff
@@ -0,0 +1,139 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.config.simplereport.DataHubConfig;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.DataHubUploadStatus;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.DynamicUpdate;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.UpdateTimestamp;;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+
+@Entity
+@DynamicUpdate
```

Comment:
I had not seen this annotation before, I like it!

Reply:
It probably messes with query caching in some cases, but it makes it much easier to debug things, among other things. :-)